source/ways-of-working/new-component/github-repo.html.md.erb

```diff
@@ -37,6 +37,6 @@ Any developer added to [hmcts org](https://github.com/hmcts) is able to create a
     </table>
     
 6. Add a branch protection rule for `master` branch with recommended settings. This is in https://github.com/[REPO]/settings/branches, then clicking "Add rule" beside Branch protection rules.
-   - Enable `Require a pull request before merging` and `Require approvals`
-   - Choose which status checks needs to be mandatory for merging PRs (this can be done only after checks like Jenkins are setup)
-   - We recommend enabling `Include administrators`.
+   - Enable `Require a pull request before merging` and `Require approvals` (minimal 1)
+   - Choose which status checks needs to be mandatory for merging PRs. The standard Jenkins check is `continuous-integration/jenkins/pr-merge` (this can be done only after checks like Jenkins are setup)
```

Comment:
```suggestion
   - Choose which status checks needs to be mandatory for merging PRs. The standard Jenkins check is `continuous-integration/jenkins/pr-merge` (this can be done only after checks like Jenkins have been run once)
```